region/maridia/inner-green.json

```diff
@@ -252,11 +262,44 @@
                 }
               ]
             },
+            {
+              "id": 3,
+              "strats": [
+                {
+                  "name": "Oasis Cross Room Jump with Screw Attack and Speed through the Top Door",
+                  "notable": true,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    "ScrewAttack",
+                    "SpeedBooster",
+                    "canMomentumConservingTurnaround",
+                    {"or": [
+                      {"adjacentRunway": {
+                        "fromNode": 1,
+                        "usedTiles": 14,
+                        "physics": ["air"]
+                      }},
+                      {"adjacentRunway": {
+                        "fromNode": 2,
+                        "usedTiles": 14,
+                        "physics": ["air"]
+                      }}
+                    ]}
+                  ],
+                  "note": [
+                    "Use Screw Attack to break the bomb block by entering from a non-water room with a spin jump, and make it all the way through the top door.",
```

Comment:
This uses exactly 14 tiles no more, no less, but it does look a tile wide.  More rises too fast to do the trick, less does not have the height to reach the door.
Using angle to break spin works better to break spin, not sure why.
The first momentumConservingTurnaround is done as early as possible, right as you pass the above tile horizontally.  The background may help tell when that happens.
The second momentumConservingTurnaround is as late as possible, just before touching the above platform.
A third momentumConservingTurnaround is likely needed on the door as it is opening.